packages/ux-capture/src/Zone.js

```diff
@@ -55,12 +56,30 @@ Zone.prototype.measure = function(triggerName) {
 		typeof window.performance !== 'undefined' &&
 		typeof window.performance.measure !== 'undefined'
 	) {
-		// check if 'end mark' was recorded before start mark - if so, end should
-		// be same as start (measured time is 0)
-		const triggerMark = window.performance.getEntriesByName(triggerName, 'mark');
-		const startMark = window.performance.getEntriesByName(startMarkName, 'mark');
-		const endMarkName =
-			triggerMark.startTime < startMark.startTime ? startMark : triggerName;
```

Comment:
looks like a simpler fix is

```js
		// check if 'end mark' was recorded before start mark - if so, end should
		// be same as start (measured time is 0)
		const triggerMark = window.performance.getEntriesByName(triggerName, 'mark').pop();
		const startMark = window.performance.getEntriesByName(startMarkName, 'mark').pop();
		// startMark may not exist as a UserTiming mark, in which case endMarkName is always the triggerName
		const endMarkName =
			startMark && triggerMark.startTime < startMark.startTime ? startMark : triggerName;
```

- avoids the value reassignment with `let`
- more succinct
- doesn't have to know about `navigationStart` as a special case, just that startMark may not exist in UserTiming (simpler mental model)

Reply:
Added additional comment about special nature on `navigationStart` to explain why would we not find a `startMark` (in case of navigationStart)